WordPress/Classes/ViewRelated/InAppUpdate/BlockingUpdateView.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+import DesignSystem
+import UIKit
+import StoreKit
+
+final class BlockingUpdateViewController: UIHostingController<BlockingUpdateView> {
+    init(viewModel: AppStoreInfoViewModel) {
+        super.init(rootView: .init(viewModel: viewModel))
+    }
+
+    @MainActor required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct BlockingUpdateView: View {
+    let viewModel: AppStoreInfoViewModel
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text(viewModel.title)
+                .font(.system(.title))
+                .fontWeight(.heavy)
+                .foregroundStyle(.primary)
+                .padding(.top, 24)
+
+            Text(viewModel.message)
+                .font(.system(.body))
+                .foregroundStyle(.secondary)
+
+            appInfoView
+                .padding([.top, .bottom], 16)
+
+            whatsNewView
+
+            Spacer()
+
+            buttonsView
+        }
+        .padding([.leading, .trailing], 20)
+        .interactiveDismissDisabled(BuildConfiguration.current != .localDeveloper) // Todo: Remove condition later
```

Comment:
I noticed this during testing and thought it was a defect. Can you define "later"?

Reply:
I initially added this for ease of testing. But I've removed it just to be on the safer side
[6e63eb4](https://github.com/wordpress-mobile/WordPress-iOS/pull/23195/commits/6e63eb4d92c19c66b941350d0f717220d3d5e0b5)